src/newgrf.cpp

```diff
@@ -6659,6 +6666,18 @@ static void SkipIf(ByteReader *buf)
 				break;
 			case 0x12: result = GetRoadTypeByLabel(BSWAP32(cond_val)) != INVALID_ROADTYPE;
 				break;
+			case 0x13: {
+					RailType rt1 = GetRailTypeByLabel(_cur.grffile->railtype_list[GB(cond_val, 0, 8)]);
```

Comment:
double indent for both these cases

Reply:
Is that somewhere in the code style? this way felt the most natural to me, otherwise the closing } would be at the same indentation level as the next `case`

Reply:
https://wiki.openttd.org/Coding_style#Control_flow yup :)